docs/aws.md

```diff
@@ -586,9 +586,9 @@ For more details of configuration, see sections [URL Connection HTTP Client Conf
 
 Configure the following property to set the type of HTTP client:
```

Comment:
we might need to update the part of EMR bootstrap actions, today it includes the url-connetion-client, which is no longer true.